### ISSUE TITLE
AdcScale -> BoardAdc bugged?

### ISSUE THREAD

**Issue by bruno-f-cruz** [bonsai-rx/ephys#8]
Hi,

When reading analog inputs using the open ephys acq board, of a signal that oscillates between 0-5V (confirmed with osciloscope and openEphys software), and converting it to SI units using AdcScale->BoardAdc, the output is not what I would expect and oscilates between ~1.8 (0V) and ~3.3 (5V). 

I am using an Intan headstage, if that is relevant.

Thanks,
Bruno

**Comment by glopesdev** [comment]
That is correct, unfortunately the ADC scale on the open-ephys system is different from the one specified in the RHD2000 eval board (the official RHD system from Intan). This happens because on the RHD2000 the analog inputs are unipolar from 0V to +3.3V while on the OpenEphys system they are bipolar with a range from -5V to +5V.

The current workaround is to use the `ConvertScale` operator to apply the correct scaling factor for SI units, which according to the [GUI source](https://github.com/open-ephys/plugin-GUI/blob/07391b0f3aa1343ef17aa0ed7efffa5737b24dd7/Plugins/IntanRecordingController/RHD2000Thread.cpp#L1015) is 0.00015258789.

You probably also need to use the `Shift` property to center the data around zero.

**Comment by glopesdev** [comment]
I will keep the issue open though as it makes sense to consider ways of addressing it.

**Comment by bfpcruz** [comment]
Awesome thanks!
